sccp/imports/scRNA.py

```diff
@@ -184,7 +184,10 @@ def ThompsonXA_SCGenes(saveXA=False, offset=1.0):
     
     ### I *believe* that padding with zeros does not affect PARAFAC2 results.
     ### We should check this though.
-    XA.values -= np.nanmean(XA.values, axis=(0,1), keepdims=True)       
+    cellCount = np.count_nonzero(~np.isnan(XA[:, :, 0].to_numpy()), axis=1)
+    XA.values /= np.reshape(cellCount, (-1, 1, 1))
```

Comment:
Did you mean to add this different normalization? What are you aiming to do here? It looks like per-condition scaling by the number of cells?

Reply:
Yes, you commented before that you wanted me to "... normalize by the total cell number with each drug...", is this not what you meant?

Reply:
I meant normalize your plots of the cell counts within each condition, not the data itself.